tests/js/spec/views/projectDetail/index.spec.tsx

```diff
@@ -13,6 +14,10 @@ describe('ProjectDetail', function () {
   beforeEach(() => {
     PageFiltersStore.reset();
     ProjectsStore.reset();
+    // @ts-ignore no-console
+    // eslint-disable-next-line no-console
+    console.error = jest.fn();
```

Comment:
why are we mocking `console.error`?

Reply:
I think there were console.error calls which failed the test, but let me try running this again

Reply:
I dropped it

Reply:
do we need `console.error `in the other tests too?

Reply:
Yeah, they end up failing otherwise 😭